packages/algorand-js/src/interpreter/opcode-list.ts

```diff
@@ -1945,3 +1875,83 @@ export class GetAssetDef extends Op {
     }
   }
 }
+
+/** Pseudo-Ops **/
+// push integer to stack
+// push to stack [...stack, integer value]
+export class Int extends Op {
+  readonly uint64: bigint;
+
+  /**
+   * Description: Sets uint64 variable according to arguments passed.
+   * @param args Expected arguments: [number]
+   * @param line line number in TEAL file
+   */
+  constructor (args: string[], line: number) {
+    super();
+    assertLen(args.length, 1, line);
+
+    let uint64;
+    if (Object.values(TxnOnComplete).includes(args[0])) { // check if string is keyof TxnOnComplete Enum
```

Comment:
Why can't we use:
```
let x = TxnOnComplete[args[0] as keyof typeof TxnOnComplete];
if (x !== undefined) ...
```

In your version we are creating a new list each time we call a constructor, and iterating over the list.

Reply:
Thanks for suggestion. I was doing this way because the above code will work for `0` as well. (which we don't want, we want `NoOp`). 
I have updated using types only:
```
let x = TxnOnComplete[args[0] as keyof typeof TxnOnComplete];
if (x !== undefined && typeof x === "number") ...
```